Longest_Palindromic_Subsequence/Longest_Palindromic_Subsequence.cpp

```diff
@@ -0,0 +1,121 @@
+/*
+    LONGEST PALINDROMIC SUBSEQUENCE
+    The task is to find the length of the longest possible subsequence
+    of a given word which is a palindrome.
+    A palindrome is a string that is same when read from any side.
+*/
+#include <bits/stdc++.h>
+using namespace std;
+#define lint long long int
+
+/*
+    Printing Longest Palindromic Subsequence
+*/
+string print_lps(vector<vector<lint> > lps_dp, lint len, string str, string rev) {
+    lint row = len, col = len;
+    string ans = "";
+    for (;row > 0 && col > 0;) {
+        if (str[row-1] == rev[col-1]) {
+            ans = str[row-1] + ans;
+            row--;
+            col--;
+        } 
+        else {
+            if (lps_dp[row - 1][col] > lps_dp[row][col - 1]) {
+                row--;
+            } 
+            else {
+                col--;
+            }
+        }
+    }
+    return ans;
+}
+
+/*
+    We use Dynamic Programming approach to solve this problem.
+    A 2D matrix M of size n*n is created where n is the length of the given
+   string. Here the row i denotes the char S[i] of the string while column j
+   denotes char R[j], where R is the reverse of the string S. Thus, M[i][j]
+   denotes the max length of longest common subsequence between the string
+    S[0...i] and R[0..j]. Hence M[n][n] will compute the longest common
+   subsequence of S[0..n] and R[0..n], which is the required answer.
+*/
+
+void lps(string str) {
+    string rev = str;
+    int len = str.length();
+    // Stroring the reverse of the string.
+    reverse(rev.begin(), rev.end());
+    vector<vector<lint>> lps_dp;
```

Comment:
I am getting this error in line 50:
`error: '>>' should be '> >' within a nested template argument list`

Please add a space between the two brackets